### PR TITLE
Resolved Bug/Sequelize Connection Refused Error - Connection.js Corrections

### PR DESCRIPTION
**Pull request by ErikPlachta**
## ERROR: Getting the below error when navigating to the API Server root  https://sheet-show.herokuapp.com/

```JSON

{"name":"SequelizeConnectionRefusedError","parent":
{"errno":-111,"code":"ECONNREFUSED","syscall":"connect","address":"127.0.0.1","port":3306,"fatal":true},"original":
{"errno":-111,"code":"ECONNREFUSED","syscall":"connect","address":"127.0.0.1","port":3306,"fatal":true}}
```

image

--- 

### ✅  Verified issue

**The models to a connection within the seeds folder that did NOT have JawsDB configuration setup.**

image

### ✅ Resolved Issue Locally

**Pointed models and seed routes to the proper path, ./config/connection.js** 

### ✅ Resolved Issue on Heroku

**Pushed to Heroku and verified working now**

image

### ✅ Pushed to GitHub

Pushed on branch **bug/SequelizeConnectionRefusedError** and making a pull-request now.

### CODE REVIEW

File: seeds/connection_sequelize.js [hunk 1]
```diff
@@ -2,18 +2,28 @@
...
+    process.env.DB_PASSWORD,
+    {
+      host: process.env.SERVER_PATH,
+      dialect: 'mysql',
+      port: 3306
+    },
+    console.log(`//-- Connection MYSQL database with npm package Sequelize success!`)
+  );
+}
 //-- exporting created sequelize obj
```
Comment (ErikPlachta) [inline seeds/connection_sequelize.js:28]: This file was actually deleted from the seeds folder,

Comment (ErikPlachta) [reply seeds/connection_sequelize.js]: Confirming I pushed with this file delete!

File: seeds/seed.js [hunk 2]
```diff
@@ -40,7 +40,8 @@ async function seedDatabase () {
 async function seedTables() {
 
   //-- Used to build SQL seed data, and erase anything that may exist
-  const sequelize = require('./connection_sequelize');
+  // const sequelize = require('./connection_sequelize');
+  const sequelize = require('../config/connection');
```
Comment (ErikPlachta) [inline seeds/seed.js:45]: Confirming seeds also pointing to proper config config files now.

**Review by ErikPlachta — COMMENTED** [review]
Confirming I've reviewed changes.

**Read for team to do a quality check**

**Review by NicaVulcan — APPROVED** [review]

**Pull request merged**